plugins/python-build/share/python-build/anaconda3-2020.11

```diff
@@ -0,0 +1,19 @@
+case "$(anaconda_architecture 2>/dev/null || true)" in
```

Comment:
This looks unrelated to the arm64 change. If we pull this file into it's own PR then it is easier to review. Apologies, I did not communicating this well on #1766.